integration-tests/src/main.rs

```diff
@@ -0,0 +1,10 @@
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
This test executes only third-party code, so it is not clear what is actually testing for us. And, it will fail if we run it on CI, because there is not nat service running on localhost.

Reply:
removed